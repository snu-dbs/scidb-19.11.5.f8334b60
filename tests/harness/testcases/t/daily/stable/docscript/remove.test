
--test
--start-query-logging

--set-format dense
store(build(<val:double>[i=0:9,10,0],1),source)  
scan(source)  -- Inserted by scan_doctor
--set-format dcsv
remove(source) 

--stop-query-logging


--cleanup

