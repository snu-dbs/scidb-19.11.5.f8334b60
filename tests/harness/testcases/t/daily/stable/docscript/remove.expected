SCIDB QUERY : <store(build(<val:double>[i=0:9,10,0],1),source)>
Query was executed successfully

SCIDB QUERY : <scan(source)  -- Inserted by scan_doctor>
[(1),(1),(1),(1),(1),(1),(1),(1),(1),(1)]

SCIDB QUERY : <remove(source)>
Query was executed successfully

